kroxylicious/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -122,6 +123,14 @@ public boolean useIoUring() {
         return useIoUring;
     }
 
+    private final String kroxylicious_inbound_downstream_messages = "kroxylicious_inbound_downstream_messages";
+
+    private final String kroxylicious_decoded_messages = "kroxylicious_decoded_messages";
+
+    private final String kroxylicious_bytes_read = "kroxylicious_bytes_read";
```

Comment:
is this a dead metric now?

Reply:
yeah, you're right, thanks! I've update them all now. I _think_ it's right to have `Metrics.summary(kroxylicious_request_size_bytes);` pre-registered only once, even though it's use twice, as the difference will be determined through the tags, right?